src/app/shared/services/courses/courses.service.ts

```diff
@@ -37,7 +37,12 @@ export class CoursesService {
     );
   }
 
-  updateItem(course: Course): void {}
+  updateItem(course: Course): void {
```

Comment:
Good practice hint: if you have this kind of solution that you have an array of entites, and you often access it by id, it's good practice to convert it to map, where key is ID and entity is value. This pattern is often used in redux when you store lists.